### PR TITLE
Implemented emailjs in contact form

### PR DESCRIPTION
**Pull request by capsule11**

### CODE REVIEW

File: src/pages/contact/index.jsx [hunk 1]
```diff
@@ -22,21 +24,30 @@ const Contact = () => {
...
+        {
+          publicKey: import.meta.env.VITE_PUBLIC_KEY,
+        }
+      )
+      .then(
+        () => {
+          toast.success("We will get back to you soon!");
+        },
+        (error) => {
+          toast.error("Unable to send your message",error.text);
```
Comment (Sreejit-Sengupto) [inline src/pages/contact/index.jsx:46]: Yaha pura error ko log karke fir se push karo

Comment (capsule11) [reply src/pages/contact/index.jsx:46]: koi error nhi aaraha abhi ham check kiye

Comment (Sreejit-Sengupto) [reply src/pages/contact/index.jsx:46]: > koi error nhi aaraha abhi ham check kiye

Are dev env me nahi aaraha hoga aur prod me pata kaise chalega jab error log hi nahi ho raha to?

Comment (capsule11) [reply src/pages/contact/index.jsx:46]: done

**Pull request merged**